.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: publish
+
+on:
+  push:
+    branches: main
+
+jobs:
+  nuget:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
```

Comment:
We don't have any submodules in `Lib9c` yet. Let's leave it as default. (which is `false`)

Reply:
I amended it in 32193c13f273b578d1d39b4f02fabe0399c7ab3b.